info/guide/guide-chapter-3.md

```diff
@@ -345,16 +356,76 @@ try {
 
 Line length should be less than 120 symbols. Otherwise, it should be split.
 
-If `complex property` initializing is too long, it should be split:
+If `complex property` initializing is too long, It should be split into priorities: \
+1. Logic Binary Expression (&&  ||) \
+2. Comparison Binary Expression (> < == >= <= !=) \
+3. Other types (Arithmetical and Bit operation) (+ - * / % >> << *= += -= /= %= ++ -- ! in !in etc)
 
 **Invalid example:**
 ```kotlin
 val complexProperty = 1 + 2 + 3 + 4
 ```
 **Valid example:**
 ```kotlin
-val complexProperty = 1 + 2
-+ 3 + 4
+val complexProperty = 1 + 2 +
+    3 + 4
+```
+
+**Invalid example:**
+```kotlin
+val complexProperty = (1 + 2 + 3 > 0) && ( 23 * 4 > 10 * 6)
+```
+**Valid example:**
+```kotlin
+val complexProperty = (1 + 2 + 3 > 0) &&
+    (23 * 4 > 10 * 6)
+```
+
+If long line should be split in `Elvis Operator` (?:), it`s done like this
+
+**Invalid example:**
+```kotlin
+val value = first ?: second
+```
+**Valid example:**
+```kotlin
+val value = first
+    ?: second
+```
+
+If long line in `Dot Qualified Expression` or `Safe Access Expression`, it`s done like this:
```

Comment:
Please rephrase, this text is unclear. Also, don't forget articles.

E.g.: "**A** long dot-qualified expression or a safe-access expression can be wrapped like this: ...".
Special terms can be emphasized with _italics_, and `backticks` are better reserved for inline code fragments.

Reply:
Done [dd07c31](https://github.com/saveourtool/diktat/pull/1384/commits/dd07c31fed183aa7b78c96db025ce1e7a26bb77f)